src/style.css

```diff
@@ -0,0 +1,115 @@
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  background: #fff;
+  width: 200px;
+  padding: 32px 16px 16px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+  margin-top: 40px;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  font-size: 10px;
+  color: #616070;
```

Comment:
You can declare variables for color in :root selector. This is necessary if you need to change the color, so as not to run around the entire file and look for the color that needs to be changed, it is enough to change one variable. Try to put all the colors in variables.